addons/mail/static/src/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -21,6 +21,11 @@
 
 .o_ChatterTopbar_tools {
     padding-right: var(--ChatterTopbar-padding-right, 0);
+    border-bottom: var(--ChatterTopbar-border, $border-width solid $border-color);
+
+    &_border, & .o_ChatterTopbar_center {
+        border-left: var(--ChatterTopbar-border, $border-width solid $border-color);
```

Comment:
`--ChatterTopbar-border` value should be defined at the top level, with its fallback.
Here should just be `border-left: var(--ChatterTopbar-border)`, the same for the previous declaration using border-bottom.

Reply:
Indeed, no need for the use of a fallback here, a simple overwrite does the job and saves some lines (or maybe you meant something else?)